site/de/spec/glossar.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: page
+lang: de
+lang-ref: glossary.md
+toc: true
+title: OCR-D Glossar
+---
+
+# OCR-D Glossar
+
+> Glossar von Begriffen aus dem Bereich der Bildverarbeitung/OCR und deren Verwendung im Rahmen von OCR-D
+
+Dieser Abschnitt ist nicht-normativ.
+
+## Layout und Typografie
+
+### Border
+
+Aus der [PAGE XML-Dokumentation](https://ocr-d.de/de/gt-guidelines/pagexml/pagecontent_xsd_Complex_Type_pc_BorderType.html)
+
+> Rand der eigentlichen Seite (wenn das gescannte Bild Teile enthält, die nicht zur Seite gehören)
+
+### Glyphe
+
+In OCR-D ist eine Glyphe die atomare Einheit innerhalb eines [Wortes](#wort).
+
+### Graphemhaufen / Grapheme Cluster
+
+Siehe [Glyphe](#glyph)
+
+### Lesereihenfolge (Reading Order)
+
+Die Lesereihenfolge beschreibt die logische Abfolge von [Regionen](#region) innerhalb eines Dokuments.
```

Comment:
"..., wie sie von einem Menschen gelesen wird"?

Reply:
Probably also hint that it could or could not include material from the margins, dive into tables etc.

Reply:
Danke, hab ich hinzugefügt, @kba und @bertsky. ich markiere hier mal als 'erledigt', aber kommentiert gerne nochmal, falls es nicht passen sollte